server/src/setup/cache/redis.ts

```diff
@@ -0,0 +1,6 @@
+import redis from 'redis'
+
+export const Client = redis.createClient({
+    url: `${process.env.CACHE_URL}`,
```

Comment:
redisClient let's us inject Redis as an env variable (in-case we isolate Redis to it's own container/cluster or a separate node altogether in the future).